### PR TITLE
Add CABundle/CAProvider to Akeyless provider

### PR DESCRIPTION
**Pull request by renanaAkeyless**
## Problem Statement

What is the problem you're trying to solve?
Added CABundle/CAProvider to Akeyless provider forcustomers how is using private/self-signed certificate

## Related Issue

Fixes #...

## Proposed Changes

How do you like to solve the issue and why?

## Checklist

- [ ] I have read the [contribution guidelines](https://external-secrets.io/latest/contributing/process/#submitting-a-pull-request)
- [ ] All commits are signed with `git commit --signoff`
- [ ] My changes have reasonable test coverage
- [ ] All tests pass with `make test`
- [ ] I ensured my PR is ready for review with `make reviewable`

### CODE REVIEW

**Comment by renanaAkeyless** [comment]
@moolen
Hi, can you take a look at my PR?
Thanks!

**Review by moolen — COMMENTED** [review]
looks good to me. Did you run manual tests that the core functionality still works after the sdk upgrade?

Can you please also update the `stability-support.md` [file](https://github.com/external-secrets/external-secrets/blob/main/docs/introduction/stability-support.md#provider-feature-support), find by name/tag is now supported with this PR :pray: .

File: pkg/provider/akeyless/akeyless.go [hunk 1]
```diff
@@ -273,3 +372,109 @@ func (a *Akeyless) GetSecretMap(ctx context.Context, ref esv1beta1.ExternalSecre
...
+		MinVersion: tls.VersionTLS12,
+	}
+	client.Transport = &http.Transport{TLSClientConfig: tlsConf}
+	return client, nil
+}
+
+func (a *akeylessBase) getCACertPool(provider *esv1beta1.AkeylessProvider) (*x509.CertPool, error) {
+	caCertPool := x509.NewCertPool()
+	if len(provider.CABundle) > 0 {
+		pem, err := base64.StdEncoding.DecodeString(string(provider.CABundle))
```
[the remark this answers is not in the document]
Comment (renanaAkeyless) [reply pkg/provider/akeyless/akeyless.go]: done

File: pkg/provider/akeyless/akeyless.go [hunk 2]
```diff
@@ -273,3 +372,109 @@ func (a *Akeyless) GetSecretMap(ctx context.Context, ref esv1beta1.ExternalSecre
...
+		case esv1beta1.CAProviderTypeConfigMap:
+			cert, err = a.getCertFromConfigMap(provider)
+		default:
+			err = fmt.Errorf("unknown caprovider type: %s", provider.CAProvider.Type)
+		}
+
+		if err != nil {
+			return nil, err
+		}
+		pem, err := base64.StdEncoding.DecodeString(string(cert))
```
[the remark this answers is not in the document]
Comment (renanaAkeyless) [reply pkg/provider/akeyless/akeyless.go]: done

**Comment by renanaAkeyless** [comment]
Hi @moolen 
I have tested manually the core functionality
updated the stability-support.md
and rebase + resolve conflicts
thanks!

**Review by moolen — APPROVED** [review]
Good stuff! Thanks for updating/maintaining/adding features for this provider :+1:

**Pull request merged**